java/arcs/android/storage/service/IStorageServiceNg.aidl

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.android.storage.service;
+
+import arcs.android.storage.service.IMessageCallback;
+import arcs.android.storage.service.IStorageChannel;
+import arcs.android.storage.service.IStorageChannelCallback;
+
+/**
+ * Responsible for opening up communication channels to/from the storage service for proxy messages.
```

Comment:
Maybe worth elaborating a little here how it's different from IStorageService, and how it's not ready to be used yet.

Reply:
Wrote a quick blurb, what do you think?

Reply:
Really good, thanks!